code/modules/library/skill_learning/job_skillchips/psychologist.dm

```diff
@@ -1,7 +1,7 @@
 /obj/item/skillchip/job/psychology
 	name = "HYPERG1G4 skillchip"
 	desc = "Learn to bend the abyss to your will."
-	auto_traits = list(TRAIT_SUPERMATTER_SOOTHER, TRAIT_SUPERMATTER_MADNESS_IMMUNE)
+	auto_traits = list(TRAIT_SUPERMATTER_SOOTHER, TRAIT_MADNESS_IMMUNE)
```

Comment:
you really ok with psychologists being madness immune?

Reply:
tbh yeah, they are more interesting that way
to me is fun to think that someone roleplaying as Sigmund Freud is immune to madness